--- pam-abl-0.4.3/conf/pam_abl.conf.pld	2012-03-19 00:09:32.258679676 +0100
+++ pam-abl-0.4.3/conf/pam_abl.conf	2012-04-01 13:29:54.260171577 +0200
@@ -1,10 +1,13 @@
 # /etc/security/pam_abl.conf
 # debug
-host_db=/var/db/abl/hosts.db
+db_home=/var/lib/abl
+
+host_db=/var/lib/abl/hosts.db
 host_purge=2d
 host_rule=*:10/1h,30/1d
-host_blk_cmd=iptables -I INPUT -s %h -j DROP
-user_db=/var/db/abl/users.db
+#host_blk_cmd=/usr/sbin/iptables -I INPUT -s %h -j DROP
+
+user_db=/var/lib/abl/users.db
 user_purge=2d
 user_rule=!root:10/1h,30/1d
-user_clr_cmd=echo This is a pointless command! user: %u host: %h service: %s >/dev/null
+#user_clr_cmd=echo This is a pointless command! user: %u host: %h service: %s >/dev/null
